### ISSUE TITLE
Backport 1.8: argmin / findmin incorrect behaviour

### ISSUE THREAD

**Issue by matbesancon** [JuliaSparse/SparseArrays.jl#239]
Exactly the same issue as https://github.com/JuliaSparse/SparseArrays.jl/pull/93 since the other one was closed :/

```julia
julia> using SparseArrays

julia> direction = spzeros(3); direction[1] = 2
2

julia> argmin(direction) === nothing
true
```

it is a fairly major bug IMO, I'm surprised 1.8 wasn't tested for breaking changes on packages?

**Comment by matbesancon** [comment]
For completeness, this was resolved in SparseArrays main but not integrated in 1.8, should probably have a patch release?

**Comment by matbesancon** [comment]
ping @dkarrasch

**Comment by dkarrasch** [comment]
The fix has been backported to v1.8.4.
